CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 ## next
   * Switch the locally generated index.html file to browse the generated OpenAPI docs to use `elements` instead of `reDoc`
   * Spruce up the initial Gemfile for the generated example app
+  * Fix Praxis::Mapper ordering code, to not prefix top level columns with the table name, as this completely confuses ActiveRecord which
```

Comment:
It'll be interesting to see what the AR contributors say about this one. I vaguely recall a similar issue we discovered a few years back, where using SQL fragments that specify table names, causes AR to switch to some sort of weird compatibility mode where it falls back to join-based eager loading.